packages/e2e/lib/e2e.dart

```diff
@@ -36,6 +37,11 @@ class E2EWidgetsFlutterBinding extends LiveTestWidgetsFlutterBinding {
 
   final Completer<bool> _allTestsPassed = Completer<bool>();
 
+  /// Map which stores failure details.
```

Comment:
nit: you wouldn't lose any meaning if you removed "Map which"

Reply:
done.